flogging/flogging.py

```diff
@@ -222,8 +222,9 @@ def flush(self):
 
 def setup(
     level: Optional[Union[str, int]] = None,
-    structured: bool = False,
+    structured: bool = None,
```

Comment:
```suggestion
    structured: bool = os.getenv("FLOG_STRUCTURED", False),
```

Reply:
I also added the `os.getenv("LOG_LEVEL", "INFO")`,  in the function signature. Is that ok?

Reply:
It turns out that this is a bad practice (or at least it makes the linter fail). I can either assign that to a module level variable, or evaluate the environment variables inside the function like I was doing before.

Given that the env variables are not going to change during the execution of the script, to me it makes more sense to create module variables, but I ok with any option.

Reply:
Can you please show the exact error message

Reply:
Sure! https://github.com/FragileTech/flogging/runs/1825440192?check_suite_focus=true#step:5:25

Reply:
OK, this is a false positive then. We should `# noqa: B008` the lines.